.buildkite/pipeline.yml

```diff
@@ -94,6 +94,28 @@ steps:
       - github_commit_status:
           context: iOS Unit Tests
 
+  - label: React Native Editor Android Tests
+    command: |
+      npm ci --prefer-offline --no-audit --no-progress
```

Comment:
I think for these tests we can do:

```
npm ci --prefix gutenberg --prefer-offline --no-audit --no-progress --ignore-scripts
```

Since we only need Gutenberg's dependencies, this will avoid checking for i18n things, Jetpack dependencies and more and run faster. This is not a blocker but thought it'd be nice to try to reduce the time it takes to install all of the dependencies.